src/pages/jdbc-create.html

```diff
@@ -129,7 +129,7 @@ <h1>
                 <br/>
                 <query-editor nostorage notabs notoolbar-saved notoolbar-copy nostatus nocount norun
                               notoolbar-inference
-                              notoolbar-same-as callback-on-change="updateDirty">
+                              notoolbar-same-as callback-on-change="updateDirty" column-resize-width="1000">
```

Comment:
Is there a reason for having this parameter at all? Why don't you just set it inside the directive's controller by reading it from a constant or environment variable.

Reply:
Yes, there is. In some place we don't need to turn off column resizing, because everything look like good.
 For the value I thought that is better to be configurable because in some place the width can be different.
 But may be you are right that we do not need this. I will remove the value and add it in constant.